src/Resources/app/storefront/src/scss/_flags.scss

```diff
@@ -0,0 +1,21 @@
+$countryCodes: ad, ae, af, ag, ai, al, am, ao, aq, ar, as, at, au, aw, ax, az, ba, bb, bd, be, bf, bg, bh, bi, bj, bl, bm, bn, bo, bq, br, bs, bt, bv, bw, by, bz, ca, cc, cd, cf, cg, ch, ci, ck, cl, cm, cn, co, cr, cu, cv, cw, cx, cy, cz, de, dj, dk, dm, do, dz, ec, ee, eg, eh, er, es-ca, es-ga, es, et, eu, fi, fj, fk, fm, fo, fr, ga, gb-eng, gb-nir, gb-sct, gb-wls, gb, gd, ge, gf, gg, gh, gi, gl, gm, gn, gp, gq, gr, gs, gt, gu, gw, gy, hk, hm, hn, hr, ht, hu, id, ie, il, im, in, io, iq, ir, is, it, je, jm, jo, jp, ke, kg, kh, ki, km, kn, kp, kr, kw, ky, kz, la, lb, lc, li, lk, lr, ls, lt, lu, lv, ly, ma, mc, md, me, mf, mg, mh, mk, ml, mm, mn, mo, mp, mq, mr, ms, mt, mu, mv, mw, mx, my, mz, na, nc, ne, nf, ng, ni, nl, no, np, nr, nu, nz, om, pa, pe, pf, pg, ph, pk, pl, pm, pn, pr, ps, pt, pw, py, qa, re, ro, rs, ru, rw, sa, sb, sc, sd, se, sg, sh, si, sj, sk, sl, sm, sn, so, sr, ss, st, sv, sx, sy, sz, tc, td, tf, tg, th, tj, tk, tl, tm, tn, to, tr, tt, tv, tw, tz, ua, ug, um, un, us, uy, uz, va, vc, ve, vg, vi, vn, vu, wf, ws, xk, ye, yt, za, zm, zw;
+
+.language-flag {
+    display: inline-block;
+    height: 12px;
+    width: 16px;
+    border: 1px solid $gray-500;
+    margin-top: 2px;
+    text-indent: 100%;
+    white-space: nowrap;
+    overflow: hidden;
+    background-position: 0 0;
+    background-repeat: no-repeat;
+    background-size: 100%;
+
+    @each $countryCode in $countryCodes {
+        &.country-#{$countryCode} {
+            background-image: url("#{$sw-asset-public-url}/bundles/swaglanguagepack/static/flags/#{$countryCode}.svg");
+        }
+    }
```

Comment:
@NiklasLimberg Bleh. That solution is not that nice. That means that everytime you load the storefront with the language pack installed, even when just using no languages, EVERY svg will be loaded.

I was not sure what happend, therefore I looked into the commits and this file was accidentally be deleted. I guess this should be the chance to create some dynamic flag loading to avoid this overhead.

Reply:
The background image only get's loaded when the CSS selector matches. So no not every flag is loaded every time. That was the whole point why I didn't just reintroduce the old SCSS file. The old SCSS file had just a huge list of hard coded svg flags https://github.com/shopware/SwagLanguagePack/commit/842298a708863295cf840099e2a964b5bb105f26#diff-ac9035fbb1392bc8e5565c88f0f9e2f2be277a02d50a193c6acf13ca67bcfd7fL23